lib/facet_name_converter.rb

```diff
@@ -0,0 +1,35 @@
+# class for converting from alexandria convention names to HCA or TIM metadata model names (i.e. columns, not individual values)
+# this is currently for PoC work on XDSS - eventually this will be replaced by an onotology server that can handle
+# conversions programmatically
+class FacetNameConverter
+  # map of alexandria metadata convention names to HCA 'short' names
+  # not fully namespaced as this is what is indexed in TDR
+  ALEXANDRIA_TO_HCA = {
+    biosample_id: 'biosample_id',
+    cell_type: 'cell_type',
+    donor_id: 'donor_id',
+    disease: 'disease',
+    library_preparation_protocol: 'library_construction_method',
+    organ: 'organ',
+    organism_age: 'organism_age',
+    sex: 'sex',
+    species: 'genus_species',
+    study_name: 'project_title',
+    study_description: 'project_description'
+  }
```

Comment:
What's the use case for storing equivalent mappings?

Naively, storing only non-equivalent mappings (e.g. `species: 'genus_species'`) seems more maintainable.

Reply:
This is really just super quick-and-dirty, and I did the equivalent mappings just to ensure a return for a "known good".  I'm open to any suggestions on how to do this more robustly, in the absence of an ontology server to do conversions for us.